### DISCUSSION TITLE
What is "You should receive an email from us in less than 31 days" mean

### DISCUSSION THREAD

**Discussion by emoemperor** [community/community#128804] [GitHub Education] [answered]
### Hi everyone,

![image](https://github.com/community/community/assets/172920309/8f9d4e4d-d936-41e1-be02-5402b78d7afa)
What does this mean, do I still have a chance to apply?

**Comment by samus-aran** [comment, marked as answer]
That seems extreme 😅  Usually it's 72 hours but at the moment I recommend you raise a discussion if your benefits haven't been granted after your approval within 5 days 👍
